chart/kubeapps/values.yaml

```diff
@@ -139,77 +192,144 @@ apprepository:
+    ##
     limits:
       cpu: 250m
       memory: 128Mi
     requests:
       cpu: 25m
       memory: 32Mi
-  nodeSelector: {}
-  tolerations: []
+  ## Affinity for pod assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
   affinity: {}
+  ## Node labels for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+  ## Tolerations for pod assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: {}
 
-# Hooks are used to perform actions like populating apprepositories
-# or creating required resources during installation or upgrade
+## Hooks are used to perform actions like populating apprepositories
+## or creating required resources during installation or upgrade
+##
 hooks:
+  ## Bitnami Kubectl image
+  ## ref: https://hub.docker.com/r/bitnami/kubectl/tags/
+  ##
   image:
     registry: docker.io
     repository: bitnami/kubectl
     tag: 1.12.10-r23
-  nodeSelector: {}
-  tolerations: []
+  ## Affinity for hooks' pods assignment
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
   affinity: {}
+  ## Node labels for hooks' pods assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+  ## Tolerations for hooks' pods assignment. Evaluated as a template.
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: {}
 
-# Tiller Proxy is a secure REST API on top of Helm's Tiller component used to
-# manage Helm chart releases in the cluster from Kubeapps. Set tillerProxy.host
-# to configure a different Tiller host to use.
+## Tiller Proxy is a secure REST API on top of Helm's Tiller component used to
+## manage Helm chart releases in the cluster from Kubeapps. Set tillerProxy.host
+## to configure a different Tiller host to use.
+##
 tillerProxy:
   replicaCount: 2
+  ## Bitnami Kubeapps Tiller Proxy image
+  ## ref: https://hub.docker.com/r/bitnami/kubeapps-tiller-proxy/tags/
+  ##
   image:
     registry: docker.io
     repository: kubeapps/tiller-proxy
     tag: latest
+  ## Tiller Proxy service parameters
+  ##
   service:
+    ## HTTP Port
+    ##
     port: 8080
   host: tiller-deploy.kube-system:44134
+  ## TLS parameters
   tls: {}
-    # ca:
-    # cert:
-    # key:
-    # verify: false
-  # https://github.com/kubeapps/kubeapps/issues/478#issuecomment-422979262
+  #  ca:
+  #  cert:
+  #  key:
+  #  verify: false
+  ## It's possible to modify the default timeout for install/upgrade/rollback/delete apps
+  ## (Default: 300s)
+  # timeout: 300
+  ## Tiller Proxy containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
```

Comment:
this comment is now a bit confusing (because its new place)

Reply:
Do you mean this one?
```yaml
  ## TLS parameters
  tls: {}
  #  ca:
  #  cert:
  #  key:
  #  verify: false
```

Or nesting the comments with the ones for other parameters (e.g. `timeout`, `resources`)?